irs/src/index.js

```diff
@@ -2,19 +2,31 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { BrowserRouter as Router } from 'react-router-dom';
 import rootReducer from './reducers';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
 import thunk from 'redux-thunk';
 import logger from 'redux-logger';
+import * as types from './actions/actionTypes';
+
 // Style
 import 'bootstrap/dist/css/bootstrap.min.css';
 
+const middlewareToSaveUserToken = store => next => action => {
+  if (action.type === types.LOGIN) {
+    localStorage.setItem('token', action.payload);
+  }
+  next(action);
+};
+
 const store = createStore(rootReducer, applyMiddleware(thunk, logger));
```

Comment:
you're going to be doing a wide variety of action types with Redux, it's not required but you might be better off wrapping this in a redux devtools extension so you can use the devtools to debug potential issues: a reminder of the different ways it can be implemented here: https://github.com/zalmoxisus/redux-devtools-extension